src/pages/components/sheet.mdx

```diff
@@ -0,0 +1,169 @@
+  - chrome/src/elements/sheet/components/Header.tsx
+  - chrome/src/elements/sheet/components/Footer.tsx
+  - chrome/src/elements/sheet/components/FooterItem.tsx
+  - chrome/src/elements/sheet/components/Close.tsx
+---
+
+<Usage>
+  <Use>
+    <ul>
+      <li>To display additional information that aids the primary view</li>
+    </ul>
+  </Use>
+  <Misuse>
+    <ul>
+      <li>
+        For an experience where the primary view is obscured to display supplementary information or
+        actions, use a [Drawer](components/drawer)
+      </li>
+    </ul>
+  </Misuse>
+</Usage>
+
+## How to use it
+
+### Default
+
+A typical usage of a Sheet component.
+
+import SheetDefault from '../../examples/chrome/SheetDefault.tsx';
+import SheetDefaultCode from '!!raw-loader!../../examples/chrome/SheetDefault.tsx';
+
+<CodeExample code={SheetDefaultCode}>
+  <SheetDefault />
+</CodeExample>
+
+### Footers
+
+Sheet footers comes in three variations: no footer, footer with an [Anchor](components/anchor),
+and footer with [Buttons](components/button).
+
+import SheetFooters from '../../examples/chrome/SheetFooters.tsx';
+import SheetFootersCode from '!!raw-loader!../../examples/chrome/SheetFooters.tsx';
+
+<CodeExample code={SheetFootersCode}>
+  <SheetFooters />
+</CodeExample>
+
+### Headers
+
+Sheet headers come in three variations: no title, title, and title with description.
+
+import SheetHeaders from '../../examples/chrome/SheetHeaders.tsx';
+import SheetHeadersCode from '!!raw-loader!../../examples/chrome/SheetHeaders.tsx';
+
+<CodeExample code={SheetHeadersCode}>
+  <SheetHeaders />
+</CodeExample>
+
+### Placement
+
+Sheets can be placed at the start or end of the container. End is the default placement.
+
+import SheetPlacement from '../../examples/chrome/SheetPlacement.tsx';
+import SheetPlacementCode from '!!raw-loader!../../examples/chrome/SheetPlacement.tsx';
+
+<CodeExample code={SheetPlacementCode}>
+  <SheetPlacement />
+</CodeExample>
+
+### Size
+
+Sheets default to 380px in width. You can customise the width based on your intended
+sexperience to any size using pixels or our grid system.
+
+import SheetSize from '../../examples/chrome/SheetSize.tsx';
+import SheetSizeCode from '!!raw-loader!../../examples/chrome/SheetSize.tsx';
+
+<CodeExample code={SheetSizeCode}>
+  <SheetSize />
+</CodeExample>
+
+## API
+
+### Sheet
+
+<Component components={props.data.mdx.components} componentName="sheet" />
+
+The Sheet component provides state and accessibility attributes to its children.
+
+<PropSheet components={props.data.mdx.components} componentName="sheet" />
+
+### Sheet.Body
+
+The Sheet Body component is placed in the [`Sheet`](#sheet) component.
+This component requires accessible functionality to make the
+element focusable, if keyboard scrolling is needed.
+
+<Component components={props.data.mdx.components} componentName="sheet.body" />
+
+<PropSheet components={props.data.mdx.components} componentName="sheet.body" />
```

Comment:
It seems the any component other than `Sheet` doesn't have a given `Extends HTMLAttributes<HTMLElement>` attributed. Is there something missing on my end on why this is missing?

Reply:
Example:
<img width="789" alt="Screen Shot 2022-01-21 at 3 06 20 PM" src="https://user-images.githubusercontent.com/12474067/150593344-76a8f59f-c6e5-45ba-9ce4-d74a813619e1.png">

Reply:
In its current form, you are responsible for fulfilling the JSDoc API for subcomponents. See https://github.com/zendeskgarden/website/blob/main/src/examples/accordions/components.ts (data) and https://github.com/zendeskgarden/website/blob/main/src/pages/components/accordion.mdx (usage) for details.

With the follow-on to https://github.com/zendeskgarden/react-components/pull/1273, I'm expecting to automate the data gathering for subcomponents, just as it is for top-level element components.